.eslintrc.js

```diff
@@ -1,6 +1,5 @@
 module.exports = {
   extends: [
-    '@mate-academy/eslint-config-react-typescript',
```

Comment:
`'@mate-academy/eslint-config-react-typescript',` removing this line probably causes lint errors